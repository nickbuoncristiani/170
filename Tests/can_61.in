61
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61
6 30 12 38 24 61 11 48 3 42 17 45 27 29 2 46 40 56 22 8 52 47 50 59 49
41
x 1.526 1.08481 1.67513 1.05693 1.75941 1.17337 1.37279 x 1.56047 x 1.21514 1.75701 x x x x 1.46894 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.526 x 1.19023 1.09144 1.25779 1.62863 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.08481 1.19023 x 1.24266 1.90587 1.53329 1.00186 x x x x 1.33996 x x 1.53413 x 1.85276 1.58985 1.5792 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.67513 1.09144 1.24266 x 1.18378 1.01375 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.05693 1.25779 1.90587 1.18378 x 1.69248 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.75941 1.62863 1.53329 1.01375 1.69248 x 1.10144 x x x x 1.08047 x x x x x 1.16514 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.17337 x 1.00186 x x 1.10144 x 1.21774 1.30895 1.32503 1.68005 1.02986 1.32998 1.08253 1.7999 1.13412 1.51817 1.85954 1.27584 1.40844 1.8856 1.35141 1.16863 1.7824 1.84975 1.57813 1.47052 1.20332 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.37279 x x x x x 1.21774 x 1.46602 1.9274 1.29359 1.59551 1.84796 1.67722 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.30895 1.46602 x x 1.79155 x 1.90238 1.29598 x x x x x x x 1.90656 x 1.1604 1.20307 x x x x 1.92481 1.93315 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.56047 x x x x x 1.32503 1.9274 x x x 1.95637 1.18154 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.68005 1.29359 1.79155 x x x 1.52525 1.81082 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.21514 x 1.33996 x x 1.08047 1.02986 1.59551 x 1.95637 x x 1.24679 x x x x 1.30892 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.75701 x x x x x 1.32998 1.84796 1.90238 1.18154 1.52525 1.24679 x 1.48576 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.08253 1.67722 1.29598 x 1.81082 x 1.48576 x x x x x x x x 1.35995 x 1.10337 1.88332 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.53413 x x x 1.7999 x x x x x x x x 1.61297 1.35723 1.95605 1.66771 1.94587 1.14104 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.13412 x x x x x x x 1.61297 x x x 1.63433 1.24717 1.27828 x 1.88233 x x 1.78783 x 1.5674 x x x x x 1.03415 x 1.25128 1.63816 x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.85276 x x x 1.51817 x x x x x x x 1.35723 x x 1.05883 1.1004 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.46894 x 1.58985 x x 1.16514 1.85954 x x x x 1.30892 x x 1.95605 x 1.05883 x 1.67504 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.5792 x x x 1.27584 x x x x x x x 1.66771 1.63433 1.1004 1.67504 x 1.53478 1.26446 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.40844 x x x x x x x 1.94587 1.24717 x x 1.53478 x 1.28026 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.8856 x x x x x x x 1.14104 1.27828 x x 1.26446 1.28026 x x 1.8906 x x 1.99139 x 1.95049 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.35141 x 1.90656 x x x x 1.35995 x x x x x x x x 1.79558 1.34983 1.15569 1.54522 1.62595 x 1.42451 1.7791 1.83621 1.57688 1.01142 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.16863 x x x x x x x x 1.88233 x x x x 1.8906 1.79558 x x 1.76879 1.74157 1.49909 1.59192 1.39342 x x 1.06131 1.52747 1.37855 1.63802 1.6425 1.7048 1.53008 1.37066 1.27957 1.85597 1.73949 1.62178 x x x x x x x x x x x x x x x x x x
x x x x x x 1.7824 x 1.1604 x x x x 1.10337 x x x x x x x 1.34983 x x 1.76236 x x x x 1.9058 1.3799 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.84975 x 1.20307 x x x x 1.88332 x x x x x x x 1.15569 1.76879 1.76236 x 1.60253 1.06794 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.57813 x x x x x x x x 1.78783 x x x x 1.99139 1.54522 1.74157 x 1.60253 x 1.77421 1.90537 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.47052 x x x x x x x x x x x x x x 1.62595 1.49909 x 1.06794 1.77421 x x 1.90022 x x 1.72582 1.5872 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x 1.20332 x x x x x x x x 1.5674 x x x x 1.95049 x 1.59192 x x 1.90537 x x x x x x x 1.41827 x 1.21321 1.94637 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.42451 1.39342 x x x 1.90022 x x x x 1.08567 1.67535 x x x x x x 1.86597 1.47959 1.63656 x 1.74906 1.00475 1.92067 x x x x x x x x x x x x x x x
x x x x x x x x 1.92481 x x x x x x x x x x x x 1.7791 x 1.9058 x x x x x x 1.24159 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.93315 x x x x x x x x x x x x 1.83621 x 1.3799 x x x x x 1.24159 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.57688 1.06131 x x x 1.72582 x 1.08567 x x x 1.92461 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.01142 1.52747 x x x 1.5872 x 1.67535 x x 1.92461 x x x x x x x 1.04251 1.04713 1.90735 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.03415 x x x x x x 1.37855 x x x x 1.41827 x x x x x x 1.1159 1.15159 1.91484 1.97515 1.1819 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.63802 x x x x x x x x x x 1.1159 x x 1.63583 1.12074 1.85264 1.71622 1.32242 x 1.38087 x x x 1.03723 x 1.01524 1.24006 x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.25128 x x x x x x 1.6425 x x x x 1.21321 x x x x x 1.15159 x x 1.02373 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.63816 x x x x x x 1.7048 x x x x 1.94637 x x x x x 1.91484 1.63583 1.02373 x 1.68613 1.90352 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.53008 x x x x x x x x x x 1.97515 1.12074 x 1.68613 x 1.78802 x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.37066 x x x x x x x x x x 1.1819 1.85264 x 1.90352 1.78802 x 1.23245 1.97037 x 1.95245 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.27957 x x x x x 1.86597 x x x 1.04251 x 1.71622 x x x 1.23245 x 1.16072 1.54788 1.87403 1.28883 1.43325 1.48304 1.78432 1.78814 1.87401 1.28777 1.76764 1.53317 1.91156 1.19038 1.16528 1.01289 x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.85597 x x x x x 1.47959 x x x 1.04713 x 1.32242 x x x 1.97037 1.16072 x 1.27324 1.74053 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.73949 x x x x x 1.63656 x x x 1.90735 x x x x x x 1.54788 1.27324 x x 1.60185 1.56292 1.87625 x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.62178 x x x x x x x x x x x 1.38087 x x x 1.95245 1.87403 1.74053 x x x x x 1.2638 x 1.96236 1.57855 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.74906 x x x x x x x x x x 1.28883 x 1.60185 x x 1.34638 1.06607 x x x x x x 1.10404 1.38664 1.82052 x 1.79734 1.1099 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.00475 x x x x x x x x x x 1.43325 x 1.56292 x 1.34638 x 1.04009 x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.92067 x x x x x x x x x x 1.48304 x 1.87625 x 1.06607 1.04009 x x x x x x x 1.83342 1.95468 1.77874 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.03723 x x x x 1.78432 x x 1.2638 x x x x 1.86817 1.95352 1.11463 1.07477 1.87959 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.78814 x x x x x x 1.86817 x x 1.88134 1.06155 1.11931 1.12993 1.90208 x 1.0935 x x 1.63658 1.12345 1.82073
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.01524 x x x x 1.87401 x x 1.96236 x x x 1.95352 x x 1.78289 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.24006 x x x x 1.28777 x x 1.57855 x x x 1.11463 1.88134 1.78289 x 1.75209 1.01358 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.76764 x x x x x x 1.07477 1.06155 x 1.75209 x 1.65493 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.53317 x x x x x x 1.87959 1.11931 x 1.01358 1.65493 x 1.15826 1.39174 x 1.91194 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.91156 x x x 1.10404 x 1.83342 x 1.12993 x x x 1.15826 x 1.00696 1.44825 1.61399 1.9223 1.37859 1.13155 1.57747 1.41973
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.19038 x x x 1.38664 x 1.95468 x 1.90208 x x x 1.39174 1.00696 x 1.10626 1.04246 x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.16528 x x x 1.82052 x 1.77874 x x x x x x 1.44825 1.10626 x x 1.83333 1.88231 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.01289 x x x x x x x 1.0935 x x x 1.91194 1.61399 1.04246 x x x x 1.50773 1.60686 1.20049
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.79734 x x x x x x x x 1.9223 x 1.83333 x x 1.65842 x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.1099 x x x x x x x x 1.37859 x 1.88231 x 1.65842 x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.63658 x x x x 1.13155 x x 1.50773 x x x 1.1048 1.06911
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.12345 x x x x 1.57747 x x 1.60686 x x 1.1048 x 1.35561
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.82073 x x x x 1.41973 x x 1.20049 x x 1.06911 1.35561 x
