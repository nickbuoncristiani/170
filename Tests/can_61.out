41 29 22 24 7 27 22 30 9 7 3 2 6 12 8 11 7 23 40 47 48 59 61 48 40 49 50 52 56 40 42 45 46 29 41
25
29 29
22 22
24 24
7 17
27 27
30 30
3 3
2 2
6 6
12 12
8 8
11 11
23 38
40 40
47 47
48 48
59 59
61 61
49 49
50 50
52 52
56 56
42 42
45 45
46 46
